src/styles/card.scss

```diff
@@ -0,0 +1,101 @@
+@import "variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  padding: 15px 14px;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 15px auto 45px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 15px;
+  }
+
+  &__code {
+    font-size: 11px;
+    margin-top: 5px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 15px;
+  }
+
+  &__feedbacks {
+    font-size: 11px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 20px;
+  }
+
+  &__prop {
+    font-size: 11px;
+    line-height: 13px;
+
+    margin-bottom: 1px;
+  }
+
+  &__value {
+    font-weight: bold;
+    font-size: 16px;
+  }
+
+  &__btn {
+    margin-top: 20px;
+    height: 40px;
+    font-size: 14px;
+    color: white;
+    text-transform: uppercase;
+    font-weight: bold;
+    text-decoration: none;
+    text-align: center;
+    line-height: 40px;
+
+    background: $linkColor;
+    border: 1px solid $linkColor;
+    border-radius: 5px;
+
+    &:hover {
+      color: $linkColor;
+      background: #fff;
+    }
+  }
+}
+
+.stars {
```

Comment:
each block should be in the separate file

Reply:
created stars.scss file for stars block